Modules/Core/Support/PaymentsHelper.php

```
@@ -0,0 +1,26 @@
+/**
+ * PaymentsHelper
+ * 
+ * Static helper class converted from procedural functions.
+ */
+class PaymentsHelper
+{
+    public static function get_currencies(): array
+    {
+        //retrieve the available currencies
+        $currencies    = new ISOCurrencies();
+        $ISOCurrencies = [];
```

PHPMD (2.15.0)

The variable $ISOCurrencies is not named in camelCase. (undefined)

(CamelCaseVariableName)